### ISSUE TITLE
Assertion message improvements when using .not

### ISSUE THREAD

**Issue by gdehmlow** [blainekasten/jasmine-enzyme#35]
Some of the messages for matchers are misleading when using `.not`.

For example, whether you do `expect(this.subject.find(...)).toBePresent()` or `expect(this.subject.find(...)).not.toBePresent()`, upon failure, the error message is "Expected contents to not be empty, but it is".

I think it would be more useful if the message changed depending on whether the assertion passes. For example, instead of this:

```js      
      compare(enzymeWrapper:Object) : Object {
        return {
          pass: enzymeWrapper.length !== 0,
          message: 'Expected contents to not be empty, but it is',
        };
      },
```

how about:

```js
      compare(enzymeWrapper:Object) : Object {
        var passes = enzymeWrapper.length !== 0;
        return {
          pass: passes,
          message: passes ? 'Expected contents to not be empty, but it is' : 'Expected contents to be empty, but it wasnt',
        };
      },
```

I think a similar conditional could be applied to some of the other matchers as well.

**Comment by blainekasten** [comment]
Thanks for the post! This was recently brought up in #31 and has a PR up for review in #34 that will be merged soon. I'm going to close this as duplicate. Be on the look out for a new version soon

**Comment by gdehmlow** [comment]
Wow my reading comprehension skills are limited. thanks

**Comment by blainekasten** [comment]
This is now released in 1.2.0 🎉
